ddtrace/contrib/trace_utils.py

```diff
@@ -279,26 +282,33 @@ def activate_distributed_headers(tracer, int_config=None, request_headers=None,
             tracer.context_provider.activate(context)
 
 
-def flatten_dict(
-    d,  # type: Dict[str, Any]
+def _flatten(
+    obj,  # type: Any
     sep=".",  # type: str
     prefix="",  # type: str
-    exclude=None,  # type: Optional[Set[str]]
+    exclude_policy=None,  # type: Optional[Callable[[str], bool]]
 ):
-    # type: (...) -> Dict[str, Any]
-    """
-    Returns a normalized dict of depth 1
-    """
-    flat = {}
+    # type: (...) -> Generator[Tuple[str, Any], None, None]
     s = deque()  # type: ignore
```

Comment:
not sure why we ignore here?

Reply:
hmm ... good question 🤔 not sure why that was needed and annoyingly mypy doesn't report it as unnecessary